protocol.go

```diff
@@ -1,9 +1,5 @@
 package blip
 
-// WebSocket [sub]protocol name for BLIP, used during WebSocket handshake.
-// Client request must indicate that it supports this protocol, else the handshake will fail.
-const WebSocketProtocolName = "BLIP_3a2"
```

Comment:
I think it's best to keep this, since every sub-protocol used by a caller should begin with this string, rather than hardcoding the BLIP protocol version into the caller.